<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpawnParameterWarpWidget</class>
 <widget class="QWidget" name="SpawnParameterWarpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Warp destination</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="warpDestinationComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="warpDestinationLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Warp size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="warpSizeComboBox"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Orientation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="horizontalOrientationBox">
         <property name="text">
          <string>Horizontal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="verticalOrientationBox">
         <property name="text">
          <string>Vertical</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
